core-java-modules/core-java-lang-2/src/test/java/com/baeldung/methodmultiplereturnvalues/MultipleReturnValuesUsingTuplesUnitTest.java

```diff
@@ -0,0 +1,28 @@
+package com.baeldung.methodmultiplereturnvalues;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.baeldung.methodmultiplereturnvalues.MultipleReturnValuesUsingTuples;
+import com.baeldung.methodmultiplereturnvalues.Tuple3;
+
+public class MultipleReturnValuesUsingTuplesUnitTest {
+
+    private MultipleReturnValuesUsingTuples multipleReturnValuesUsingTuples;
+    
+    @Before
+    public void setUp() {
+        this.multipleReturnValuesUsingTuples = new MultipleReturnValuesUsingTuples();
+    }
```

Comment:
I think the functions we're testing can be made `static`, so we can avoid these clunky test setups.

Reply:
ok